### DISCUSSION TITLE
What provisions of the act are in effect?

### DISCUSSION THREAD

**Discussion by FinalTwist** [MinistryFinancialServices/Virtual-Asset-Amendment-Consultation#18] [Q&A]
I've been told that some of the sections of the act haven't been commenced, does anyone have a list?

I'm told the sandbox hasn't commenced, and some of the provisions around isuances, would be nice to have a list.

**Comment by Aaron-XReg** [comment]
Here is the list of the provisions that have been commenced: https://legislation.gov.ky/cms/images/LEGISLATION/AMENDING/2020/2020-0137/VirtualAssetServiceProvidersLaw2020CommencementOrder2020SL137of2020.pdf
